### ISSUE TITLE
SourceModel package generation: handling of nt:resource below something else than a nt:file

### ISSUE THREAD

**Issue by stoerr** [ist-dresden/composum-nodes#309]
If you have, e.g., a sling:OrderedFolder with a nt:resource child, as in /content/ist/composum/assets/general/adaptTo.png/jcr:content on cloud.composum.com, that's not handled properly. Currently that writes adaptTo.png twice, which throws an exception. (I deleted that node since it's obsolete and currently blocks exporting the whole site, but I'll add the JSON to reproduce the problem.)

In https://jackrabbit.apache.org/filevault/vaultfs.html#Resource_Nodes there is a special case for that, which currently doesn't exist in SourceModel, I think. So that's a bit of an effort, especially as that node doesn't have an unique RenderingType anymore - it should be EMBEDDED for the attributes, but still have a binary file.

Todos:
- Extend SourceModelTest with that case
- adapt SourceModel.

Likely a rare case.

**Comment by stoerr** [comment]
```
// 20230810144459
// https://cloud.composum.com/bin/cpm/nodes/node.map.download.base64.d0.notype.json/content/ist/composum/cpl%3Aattic

{
  "jcr:created": "2022-03-24 20:48:10",
  "jcr:createdBy": "admin",
  "jcr:primaryType": "sling:OrderedFolder",
  "1aa7c9ac-11fd-4bf5-8ffb-cba0d73bd7fc": {
    "jcr:created": "2022-03-24 20:48:10",
    "jcr:createdBy": "admin",
    "jcr:primaryType": "sling:OrderedFolder",
    "assets": {
      "jcr:created": "2022-03-24 20:48:10",
      "jcr:createdBy": "admin",
      "jcr:primaryType": "sling:OrderedFolder",
      "general": {
        "jcr:created": "2022-03-24 20:48:10",
        "jcr:createdBy": "admin",
        "jcr:primaryType": "sling:OrderedFolder",
        "adaptTo.png": {
          "jcr:created": "2022-03-24 20:48:10",
          "jcr:createdBy": "admin",
          "jcr:primaryType": "sling:OrderedFolder",
          "jcr:content": {
            "jcr:baseVersion": "e9f2fbc7-a635-4d48-a60e-c6b341c6e428",
            "jcr:data": "iVBORw0KGgoAAAANSUhEUgAAAIgAAACICAYAAAA8uqNSAAAABGdBTUEAALGPC/xhBQAAACBjSFJNAAB6JgAAgIQAAPoAAACA6AAAdTAAAOpgAAA6mAAAF3CculE8AAABWWlUWHRYTUw6Y29tLmFkb2JlLnhtcAAAAAAAPHg6eG1wbWV0YSB4bWxuczp4PSJhZG9iZTpuczptZXRhLyIgeDp4bXB0az0iWE1QIENvcmUgNS40LjAiPgogICA8cmRmOlJERiB4bWxuczpyZGY9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkvMDIvMjItcmRmLXN5bnRheC1ucyMiPgogICAgICA8cmRmOkRlc2NyaXB0aW9uIHJkZjphYm91dD0iIgogICAgICAgICAgICB4bWxuczp0aWZmPSJodHRwOi8vbnMuYWRvYmUuY29tL3RpZmYvMS4wLyI+CiAgICAgICAgIDx0aWZmOk9yaWVudGF0aW9uPjE8L3RpZmY6T3JpZW50YXRpb24+CiAgICAgIDwvcmRmOkRlc2NyaXB0aW9uPgogICA8L3JkZjpSREY+CjwveDp4bXBtZXRhPgpMwidZAAAjbUlEQVR4Ae2dCbAVxdWAm31zQRZFlC2KawhGIlE08DRGTYz+EhU1iRSpStAk/qIVUxVNYpnFmNVsJibBMkZ+ISgqAQklRkQUowYJLnE3rqDIjuzLm/98A33t6emZO0vfx3u+e+rNm7kz3afP9Jw5ffosPW0CAVWHeg8k9EDbhPP10/UeCHugziB1RkjtgTqDpHZP/WL7WnfB1q1b1YoVK9Srr76qlixZot588021adMmherTrl071bt3b9WvXz81YMCAcL/HHnvkImnbtm3qjTfeUC+//LJ65ZVX1KpVq0LcHTt2DPEdfPDB6pBDDlHdu3fPhVcXfu+999Szzz6rnnvuuZD+xsbGkO6BAweqIUOGqEMPPVTRVlF4a2OjemLldvX8hh1qS6NS3dq1UUfs0VYN69le7dt597+/NWOQ+fPnqxkzZqhHH300fHgrV65U27dvd/Zj27ZtVbdu3UIm+ehHP6rOPPNMdfbZZ6s2bdo4y3PypZdeUtOmTVOzZ89WzzzzjFq7dq3i4dnAw+vfv7867rjj1JgxY9Rpp52m2revftv//e9/1U033aTuuece9cILLygY3QZoPuqoo9TnP/95NXbsWJWHuZ9Zs0P98sXNavaybeqdLYEw9fvY28pt9+/SRp21f0d16SGd1KA92r1/samPmMX4Bnmbgz59+nDLhTbp6EDe2kSy7rrrrmC//fbLjVskVvC1r30tEEZNxM2FW2+9NRCJlgv/Jz7xieD5559PxasvTn51c3DAjNWBmroqUNNku9OxcX7qymDwrNXB7KVbddUm39dEhj388MPqnXfeKczr69evV/fee6+z/ubNm9VPf/pTtWzZMuf1tJM7duxQf/7zn9XixYsTi0H7+PHj1euvv55YxnXhoYceUpdeemk4fLqu63MTX9mixj2xUS3ZKvyHIEsSkpxv30a9tDFQ5z+2Xt2zZJtG0aT7mjAInVUWwCGvSwzN448/rhYuXBg7n/UE+s/kyZOdxdFnrr32WgUTFoE5c+YokW6JVRcs36aueHqjCgerJMawa8sTWisj80WLNqoX1u2wr9b8t3cGoXOffvrp0oSjFK5bty6GB50jSZeJFU44cf/99yuUTxsee+wxxbUy8Ic//EHBaDZsE/Xommc3q3XbhOmzModGIk9p6aYd6rrnN0d0FX25lnvvDMIsooj4t2/y7bffDmc/5nmGiCeeeMI8VegYBdQ1hMycOdP5cPM0wvCF0mzDIyu2qQdXiCiQWUohkHp3L92qnm1iKeKdQZhNuN78vJ2yZs0atXTp0kg18L711luRc0V+ID1gEhOQSv/+97/NU4WO0Z+efPLJWN37lm1X2xrjQ2asYNIJ4at1IpgeXu6eCSZVK3veO4OsXr3aC4NwY/YwsGHDBgXj+ABbyjGNXb58uQ/UMeaDLRav8fBgRSd7CoWkCcE7gzD+upTLIvcEQ5gAw8CAPsCWTtC9ceNGH6hDY52JCPPMOp5ril3HLJ94jBTZXkIKJSJOvuCdQdKMW8lkuK9gQDMBA1eHDh3MU4WPu3TpEqlLW1h2fUCnTp0iaOCLUPVwzMoiBTP8aFuWyTK0YRaJPgHzSsHjrl27enuItmVyr732Ur169SpIWbSaGPIiJ7C4gt8H4DowActoP7GMlgYRHgO6en9kqWR5b61Hjx5qn332SW00y0Uk0Z577hkpCvMV9alEEMmP/fffP3KKtx5/UFlAEuH7seEE8a2UhbbCacP38SPlstLinUFgDh8PsWfPnuqAAw6I3AcMg/OtLCCFPvShD8XQjBo1KnYu74m+ffuqoUOHxqqd2qeD6oPzragKIXrMYeLEO6GXnyE2RmDCCe8MAnO4Oj+h/cTTDAFIIxt8PMTDDjtMHXjggTZqdcYZZyjx8cTO5zmBM9BmbOoPEIfb+IGim+woyCGiv0w4uJPq3snDUJXjhrwzCEPDsccem4MEd9EjjzwyNsRQkgeAdCkDn/70p5WtSIIP3eHiiy8ujBqd6atf/Wpi/csP7axG9pahJu9MRMqPObCjGjeocyLuWl3wziAQyltedjZzwgknOO8Z6UQ4QFEg/uS8885LrD5hwgQ1YsSIxOtJF5hdXXXVVeroo49OKqK6d2yjbjmmmzoefYRpbzVhwnVhjv/Zv4O68eiuqmPTqh8776MW/mOxeAYirgOZSnKLuTbRYYKTTz45kACjRNIkQCgYN25csO+++2bGLfpL8NnPfjYQX0siXn1BApuCSy65JBBmqopfXoTg4x//eDB9+vRA4lE0itT9ik2Nwf8u3BDsdTcu/5WBumOX2x8XPxu/5Xyv6auCq5/aGLy3NRve1EYLXmxDvZ2s4vc/Rqf//Oc/CgcY3ld8H0STYbHUgT3YHTp37hwG9AwePFh97GMfC4cnorSy2CSIInvggQfUggULwqgvLKHaUUZ9FGaUWoKFTjrppDC4J49ke/HFFxX+GYKfCBrinqAde4wwpxo2bJg6/fTTVUNDQ65gId3TiySSbMqbW9VDq7artzcHaocoou1Fpvfv0lad3LudOrdfJ3X43rtDbGgKxbZXKwZ5v4mdR/AhvhQ6GacbQEczdfVhf4DxwL9ly5bQkovIB69tENtJTf7/WHHxs0A7NhOU8TKhhiYFeHpXSbwh+47CID07idGuJoO/2Wq24yZjkGzk1Es1tx5oJnza3LqlTo/ugTqD6J6o7509UN7+60S78yR6B+P2u+++G7rp8c7yG48s11AksWkQHY6VFMUP41iesR2dg/hX2iAWhRQLFFWUX6a0e++9d2i4op0siq99O+9JBNjrGxrVu5sb1TI5xoTRWWxV+8mUdX9RJvt3a6s6FQ0CksbwKqPAkxJCOCT2GayxAwcOdBrzbPpq/bsmDELI4cSJE5VEeYc3T8oDN09Qjh0OgO8ChVIrlVhQsXTKNDOcfbg6gDyYBx98UBG3SoQZsyMYD0VVK8DUAyfMxmwG+wn2DQxtEoGubE+x2c4GYYQH3t2m7pZA4UdW71BLJNxP/tR2USI1oEx2k947qFs71dCrvTpbDFnDxb6BY64aEJY5a9asMDb2n//8Z/jCmHGwMAlKMLYgUiqYKbkMe9Xa8XKdWYxvuPLKK6vaD4T41DKXX365kyyZTQTyoFPrpuHGNiO5Ok7cnHx6zfbg2PvWBm2xRdwuNgptm0hKTdhVrouUG//4+mDbjnSbhQRdBzLlzkw/dpZTTjklNQ0k8WY8XKiJDkImWlkgaFnbS0xcxHxiWykKSDJsG0lw/Qub1aNin2ikZxg6kAhJUoHzu8ptksObXxebxruYSN2ApDv//PPV3Llz3QUcZ+UZK6LlScVASjY1eGcQ7Bx0RFlgGLFDDsGJUcwcRoq0w9CkDWpm/dfXN6qZ70jgp+Sj5AapIulY6v/eiGfgaVzXX399LBxRX6u2J19n0qRJ1Yp5v+6dQYgZJbK9LIDD9cYsWrSoLOowFdSV2DVX9I4VJDQV4I+QKFFAHpDI9eWi0NrASzNlyhT7dK7fpFS4XppcSHIW9s4gzFTsWNKcNIXFmY0wMzEBvCSBlwXwuqLjF8jQUirxRBjrrU2NztQElGo7UDrvfTB0l0kay9se5b0zCFNYH5HnzHiYtpqA/uALtx20zAzlFZnOFpYeuwjFXP7ie3EJQhJ7WaBPfOQF5aHDO4MwXSurI+gbwMZhArhdw45ZJuuxPQyKw1St2cqDLTq+7GpZ9JA3Nu70NWlaUDRxLPoATAdNCd4ZhM7wBaZtAJwwhw8JAi4y90wgqWkjE5CS/EF9mSmbqMPZGMq7D2jxOoiPTtA4bGbDVZ/HXa/xuPY2bsqU5Q3djguPL7qLWIM1XUX23iVImoUyL4G2qx5zvCtONS9eyttR7R1kBtIVu3L05c+PWup37xBlEfrETuHIj3hnDV/3n7V97wyCXyWPLyWNUBsP/hVfHY353QT8KT0kDqM0hwhvDOgaDfJBeuA+8AFHHHGEDzSZcXhnEHwIbGUBUYrEMAHmKxuwDD5w24lTGE0HS1pBWQnSUSRRiMckXI6PP/740sMjLwhRd00J3hmEN5wosbIAI9gpCDisfOTFgNuV9oDTrZQiIsPLAZJBd/heUQlCX4wcObJ0Ogghjq6cm7J9nVbfO4MgPewHm0ZA0jWGANdwIgHCSVUynx80aFBMB6Fyw74dJB5UHm5RPURmQiMlsam3Y3VCkrXKpFRAn6yv1uReXe8Mgt7gI3GKN9wVq4oLHFFbBoYPH+7EsZ/Ed1xwoGSuMU3NyyTCHLj6LxyQvCQmDjdCDYoAKzSy8mNTg3cG4QZYGrIsHH744c4AHyLeTz311NR4jqS2URYPOuggde655yYVURMGd1bn9Ouo9kQpIQuODWbRGzX1MXY1rgtzHCmZczcM7apG9U5OjYThb7vtNjVu3LgwSApU1QBd6fvf/364JOfuiAmpSdAyjik6Apf9a6+9FkZLYeDCVMymgekfke3cONNOOgPG+PCHP6wkNybMdNNlzT24cPk/8sgjoW+CBXR1yoMOEdC4Ee0MKay/ytuL9CDSrBosllSEObKG6YOyos/zYoJfRTSZMAJ2wHYiKjrJSDRIhpJhkkx92n4dJGOuQ660SHwqd999dxj4xJqvuBGgHQWaoZU+kDgQNXr0aC8Sudr9Jl2vCYOYjWF2x8lGSgKdwKaNVER8YetgdsLbBaPkNSiBC/z4gLC0ElUGgJuORidiNpQXr74HGIKF51aJGZ5IMwRGZ5Eue0nIYU/ZOiJpSgBhBzgmoZ9j+gAlms2nTakoiTVnkKKE1es1jx6oiQ7SPG6tToWPHqgziI9e/ADjqElUO/2FLoA3lpgO9jrinGOUMXQC9AemxSiq7HVkO7qD7YdxPoNd+sFK0Q02ydR0s2x47NEK5JLqIOzfRcIHu8nWU/wje4jOkBfwnhJghH4D7dAM7Uy10W2w+aBDFQVwo2CjR9Ev6B3gQ5EmDWR3g3cGoQMlIj1MeKZDUU5hDmYvdADHlNEAc6C5s2ejc9iwGBJiZ3f+Cklyvu31LWHU1nMyu1ghOa2rJUxwi8wwCNYxPe3oj6QnoFT2EOboI76WI2Q6en7/jmoE63Q4ALc8YY0kbP/rX/9SJHDrtA1o1wBDw8Q8RJacIkG8oaEhnC1VUy5ZaJe0h3nz5oVJ4czKbOYjnwe8zOY+85nPKNZL2R3gXUnlTWMBGTL7ywCGMha2tRetm/TqFjVWFrffGXG+SyKYgsE8fp8Pd4oUCBIOGiPGrKnHxb9LA+NizJKvPYSMnJd+mJm12pmeuoCZyje+8Y1wepsnrgUPLtHwBD03tS3Euw6CqORtKAtIHBeep1iKGtHABvVsMIXezIb1Ofa6rAw3L0v0OlNWG7Df8IBNSWGXSfvNvact5n/zzTeHX5vIwxy0R/Tb73//+zD9Ia39WlzzziCECdqhgkUI521js+FV4kZDbrCvZPwtzLJUos6XE71uAV+ScLVpFUv9SXoCUtQG7D8YD8sADGYOz2VwZa3bbBkEJrPD6zaJlWoJKQVIhKIgdVeL9FjJ978swDpb9gEQLf+aWI9tYLgsm1CGXuQjqt+mLe23dwbxtZg/RNtDDAyyrkzeyq6egDfetnJXYAxM9mWB+3fhIaq9rGRFuvnIC8pzj94ZBHOx6W/JQ4xd1s5+2ybqB0xSGgTFWvlOnAnoHa5kKrNM1mNXzk1ZpZ22YeIWH9UOc5QV0/pBMG6bwJsfqiDmyYLH69/3GYYYoNtX5Lmdc0N/uJimCOmu4asInqx1vEsQbB2+wJ5NbBBbh5fv6cgDW4bRxAAYxGZI43KuQ1t3ok984XYpwLmIy1nYO4PkbD+1uMsDGx0YUqunXnTMclPL57no8yWx260lbrstfntnkGpWRBcRWc91FhM3ccWlQWYy+1oZ/FhxM5n3MzRux+TSJ1hefYBtWfaBMw2Hj+6O4Mef4nrzI4Uy/rAfGNkEjnjgjNjMYvIlCQeD2A/WrJHn2I6Ypz/sc3nwmWVdwdbmdd/H3hmENxEm8QE2HoJzumBBLTvOCIq9Lccdb7iPYGvu28UMH/nIR3x0iWLB4aYE7wyCk8nOZyl6Q3ZwMgHne1tZa0VwE1zcw4o85y33EWwNzYMGDYqRRT4LL08Z0E7MMjjy1vXOIDiTfDiUeGA2g+CV7csXl2QWUhikancZXvZ3fFbDx1cq8O4mMQgB12UAKeQrQy8rHd4ZhIfqg0F0PKl9I4Nl2clSIAzSS4aXHsQBWACD2N5jq0jVn0OGDHEOVcTcsmJhGfjiF7/opW/z0BDvpTy1HWV5sD6UPXAwXNkwBC0VayobpgzZpQLX2ShLHXH39+tKHq6MMxbwISC+JVMUGFpxyycp6V/+8peVfHEit45G4DVJU2UZrMh9lRsUHS3yYAmewaXNRlQ7Rijm7xzbVlaChehQxmeOkUCIaUSpi9FOlBSDyw7tEgYM4U95U0zmBAuxQhDBQnr0kZlluBZde1E49hSe6is6x4GSFnmYBAyddUAH53qmTEeJuSDvhq9I4FzjA8z4V7gHNoByelrcv3//UHEk9/ZTn/pUmK7g6JbwFPdFOACL6JHyQEASKQ8Y1sBN34Ab5ZzMQoKEGhoa1FlnnZWKN6k9H+e9BwxBFE4pLH461QETNudwNuGA028Ye4JhGJJgBpgDCaRTINJsB8ITYZgh+SprxYG3XiyjfLt4vUgJQau6i75C8tOeMpzsI9teotx2ZQYUFxyJ/QjNxGKQlgCTsOdBQh9pCdDOQy9qm6AvCDlkY/0yrK1IC8INmVGx56XZnVATBtmdN1Rv228PeNdB/JJXx7a7e6DOILv7CTTz9r0rqcRrLN3YGE4ccgz3sW5C2ewj2fZ7JBjGUHi1rsNeK8Hs0W20wscYzobZHl0BfUfrQLFGrRPgQIFElyI2hTZN3CiTWmcqqiuAG/0G/Jp2FGDwoqiitO5O8M4gi1btUKMfWa+CMtxBj4gWOlG+9HhW/06R/vnTn/4UzgBQ6FB+mSlxTOeiQLLXoGcbPDz9IGEUFqH52c9+FlsJCUb7xz/+EUZtERXGZzpQrFFUCT2AQTTo2QZTcWwnzGZIU+BrEsccc4wuFtuzxDixr2x8FYPYEdIqwK9neDCeXmeFZHbWRGHZCwxwTc0w3pXUebKc9YnzJWjXA4NMHd5NUhTeZxA6kCnofffdF+v4PCd4qCxIy1IQJjD9PPHEEyOMYF7PcnzSSSeFTOaSUr/5zW/UddddVyhyDSb84x//qD73uc9lIcNbGe/yC77A1xEySMm9zWNICsRxWUCcuwJvsHuYUqJIO9hNyJSzAdpvuummQswBLqbYkydPttHW/Ld3BqklxYhhH2GB4KDDbSDjrSwwHNmL9IIT5iu72jJR977iZrPeZ4tiEHQM9AQfQJKTDRisygJKJ3qGDTAIUqQMQB+poE0JrZZB7IfFsOPr7XQNMTBIWUCC8oHnpoQWxSCYpn0MMXSwrYMgnXzhdg1frlyZIg+6xUe1F7nprHV4gHbEeNa6djlzOsw1bVexyxX5bUsQZl9lUzo1HS79Rl+rxb5FSRCmjq7po4+O8ckgDAUmwIy+dCd7aDTbqcVxi2IQnx3gUlJ94We4qhWgKzUltDgG0dbGsp1kR8xrq2tZvNS3I+qQemXjUTVdNt36fK32LYpBMEHbcapFO8Y2WfMAfeG240Noy1cgNy6DpoQWxSB0PME6PsB2rvHbF4O40ifwo/iAAQMG+ECTGUeLYhDe8rQos8x3LQXttxzcZQOWdfv2t2g47yufxRejaVqr7Vscg/h6y+14V4aBvn37VuuvqtcZBl14SFngWhmAqfEYNyW0KAahg20FsGhn2RIEPD46nyHQxSB8Kaps2iQhBT6+l5Onz1oUg/DW+/hmG1LItaA/C+iXBYYXF4MQ5EzUexlgOUxfym5WOrwHDBFt3kj+ie2rz0qRLieIQGMCwwBjOV95wBiFcSvrtJe6KKJME8lwIyDHBr4I8aUvfSkM5MFiiUUUu0NaG0g1mI18XPSD008/PfEhfu973wu/ZjF37txwpSBWVbSX2dI0QS80kg4KXTDHJz/5SX25yfbeA4aWSLjh9CViSSzLIMIcp0gOzGArnZ8IMpxqmNzxpxAfghmbB2kbkRiOeIAon7x5TBHZI4XSwvkwotEODAJuUhJoT1tIUZSRZjAFeEh94GHmmYLiryGajHtZsmRJ6OmFEWEMcDIcIYnKruRclpO8M0hZgur1m1cPtCgdpHl1Xeugps4greM5F77LOoMU7rrWUbHOIK3jORe+yzqDFO661lGxziCt4zkXvss6gxTuutZRsc4greM5F77LOoMU7rrWUbHOICnPmdhSX1H0Kc0060venXVpd0tWGD4I/A0ATjP8Ds0J8OcsXrxY3X///Wr27NlhlNnUqVNjzj2iy/nEB+WLRNoT6c7n6GsZAIQzk4QtzeREo7FQH6sW4P+BbhLZCUVIBHEQNQk899xzgRAXiPMsEGdXuI0dOzaQjmqS9rM0Is64QFYiDCQcAD9yuImzLJAOjVX/+c9/HpbT95J3Tz/IgwnkQcVw+zpx1VVXBRKfEvY57cnKCIE4HoPhw4dXzskLGtxwww2JTeLKbhJYsGBBIO72SsfzAI466qhA3sAmaT9LI3SeeGUjNIpXNZBc21h1WZYyUk4zVJ69vL2BSKEYbh8n/va3v4VMYNIDzRJeEJxzzjkR2mEiWfrC2WyT6SA6HkMIroCvVIAKQg8HuPKzgPRmlmKpZezsvtTCOS4SqnD11VdHwh8ITeCTrIRAiGSJrEFLwvm3v/1tZ3JXkzFIjvtrEUWHDRsWxmoQa6I3AodcUfc6IFqXY09MCisREUviGyZNmqSefPLJCNoLLrigsha9SG51xhlnRK7ztU7Wbo2BU67U4OTChQtDvUMIqIg3WeC+2Q0x8sAq9EFrv379wnHb7hJ0J/QHSaYOdRT0FIkQC+Rr4ZH64GAolaz8SjnKUk+URxtt6d8SQBVIgHSEBom/DUTxjuAWJTw2BI0aNSqQFNFIucgshsgp1rZI08oRVWnBt4hNVg9GbGk8DC9ZF2chwopQPF3X5mjO89bJg7Mvhb+ZVfDhP0L5knAQeohGnyW+lYi1a6+9NhTJ4ONNZBVojl2xpy6JwLA1cODATCkbwjjOxW3kqYWL91b7IsW8efPCkEmzc0aMGBH7tLsoquHqzXyuVQNf5mRpLlbKroBmF5m2hVquxGwGEk6XuKHVX3HFFYEwgK5a2QsTBGeeeWaA8mXjMWcG0njI4bYEYRYhi8DF6tr0QIOMp4GMtZW2OZDs/2DcuHGBhP6l4uCNkgj24He/+10gto4KDpRUW4JoWvX+4osvrpR3Hdxxxx2Rt5d6MpSEtLnK63O0fdFFFwUS3Ozse/pTGDr4yle+EshLpKvF9hdeeGGs/V/96lexcpy48sorY2W/+c1vRspWZjESexlOu3RHVNt/61vfiiCS+MpA1haPNZiGx2YQiQWNice0+pdeemlkmixvQGymlFafa7/4xS8q95GFQWDANCjCIAw1p5xySua+a2hoCCReNkYG9ItdJYJHYnADsYXEynKCmYvoR5HyRx99dCCxvpXyFSUVkYlhJStMmTIlXB5Sl+ez4z6+DZuHBpQq1gTTwHCQpz71hEHCIU3jqLYnCNo3zJgxQ82ZMyczWoaR3/72t7HyDOP28ld8EMBezVFX5NMldp6N2KtCFUGXqeggJBKRt2GP2/xmbQusb2bnM5XCQqfHcayPNjAeo7EDRKCjWwhr2sUqv5mCuZahTKKB9UWhg5wTQFtoKwjlgBwY9A2uoR8xxptrdbDuF587T9JpmJV84QtfCKPhmY3wuQ/fQBqHDVg40XG4d+5TpEOkCEyCK8A0FbD2qvmMqMBsK2nqThvMaMyPNWMhRi/hPFBhEBr69a9/HVlmEuJQ6CCQ9T/N78bT4VzXoFMC9G+uydgX5nNwjjVIR48eHd6ULmPvaSuJBkz0J598cvh5Dl3PbF+fs/dMJe+8884w/QEGkTFa/f3vf68Us++jcmHXAZ14zTXX1GQ6qtsypaA+d9lllyk27vHee+9V5513nr4U7mEOmMFkENdEQD/oSGXjx9ChQ9Vf//pX44xSTz31VOV3hUE4Q2fZvhHENmKr2so2roeFXYAOBrJmhCXRkJZkFDaQ8I9EKTOLzpUwlVA1PI3EMyVOWtmi11x9h/TVtGopbOK36zB7s9dBQ3JU+wwaQ5ANCALumzYiDEJBnDhwENOdRYsWheIHsVyNQexG+G2KO97erEBCEctUi+0kpIFxsSgN3Ch0IJ2APHRkpbcW5cy+M4+T2tJJZOZ1hsc0kwRlMVvASOYIgKmB3wz5FQbBbvDDH/5Q/eUvf/H2nXmT2CzHvKk/+MEPdisNWehsjmV4oOZDhkb0Spdl16Qf6YR0R43QwHOIMQjrgGMQcoEWZ2kKpqte3nNpNIALOmpNQ16am3N59BNTR3HRipRwKbG6n8NpLiJs1qxZsfp8ZQCnz+23317zb6Yl0YAVECXx1ltvrfgSYoTWTzh7AEWWLQ0YOVw6lhYK4RCDOEEZNYFE5IkTJyrmygAzkloCN2KvPIiIvPHGGxWLr3ATEoNRSxKcuOkorb84CzSTk0gCNhPQS0wzgHlNHzO0UM4ETAPa3hMxlJmFaMye0ZjXsxyboss8dtXlQTCDMQEi9RiqRZ55PcsxOM0HbB5nqc/Lg9JGlj/T0aZScvUDgsZqfUcZl76BP4xJRxpw3dZdWFFAM1tFSbWRMGuZP3++GihOJsSQa3lpu475mweKAUg3xJcK8nYu7UIDsxpuVofOme1UO+ahYoPRzGfacqgLnWnMhyGNb91yH5TjezXiw6k6tlejq9p1ZpDagIYr3gabZhgKiykONw1MfVnbfeTIkfpUbO+yfhMGGRliqGU3yLLXEhIYvn08WHss45xZR9s7TAq++93vVjoSHcNmELM+9ezrMAgLuqBouWigvo3DbJ9jpsp64RXK2vdhDiEc2/j4bS5/PXPmTPXjH/+4tHQ16XR5hRneb7nllvBB2W84dVmfxJaGfJ3KBu5fHHz26fA39+aygGu1gkKhTEeEuYYTHirE2Z1KRaZG5oIpvFk2QABczGY/fMryVmtO5WZdxjTqJdGAIcnFmCYdJg2u+2DFIt2xDGcuw5GJDzrB6ROS3nD6zcUctM0nyuwZCi58LSk1fdizeNFcwLBpuvspg2GRFY0qIDcbgphbY549KRTx9Jm/cfmbIEpkgCvcLFPtePz48SaKQKLHc3ljv/71rwfCxBUc8qmyXO0LcwUiESr1ORBxHgjDJOKRdcwi3s5IZfnBPdj3LR2e6u6n78TfE6tn49G/haGcQUwi6QLxO0XwECoh0WU2meFv+bp4rL/F9B4JZKroINj64Ug+m4VjSL/ZQlQFBGvo/GIdLnG1V85zgBTCj4JziGkxOosLB2WRCoTaEwdpwpgxY8I3ANGK3uGqDw3oA4TMXXLJJbE3xsTHMVLB9lhyHqkhDB2+ifzWwCfWWUMMnYk3z6RB041CmATMuPBbofvwNlMHH5TWxVz16DuJRAsDdfAbMfMw29V10DMYLidMmOD0DaFcIllwrGpAVcCXA102kNbBKGECzlJzZEBcxkAU1CBpE7EXK+86kVSf89LxriqRc5RJwpFEg0uCSIcGxJmY+LK0HyEm5w8Zyiq0521LGKpS177/pPs2ybvrrrsiEkQefoDEQUqZIGaNQHSNSFlh4lCCmuWcDGIWaEnHLgaREIbIMNSS7qcIrTx4hgkYQ288eNJOTJg+fXogEq5ShrINEogkOo9ZLIgaHqTUBxHkjj+It+W8J4ZUCV2MXMPIyLCtgWGF3wx/GhgOqWfaX7jWKhhEd0Jr2RPgJOGckdudNm1aJXBcpEksgg3dxvVN3g8Ug5hvhO4dpoqtDZAi2GqYsmpgSstHnUWvCZ2yKK8a8Oj+6Ec/clpsK7MYXbgl7wmOkXE0DFdgFsDGm2EblFryPWalnfv+yU9+Es6O9EvC7IvhhTBRZnb0D7Or73znOzGJo9v5wC2ky5thOv2IV7WNR/rmW8Oe6baeymJUZEotM6uK2wKdQ0euufoDBonm6LlK1c+12h74f3e1/uYvMii7AAAAAElFTkSuQmCC",
            "jcr:isCheckedOut": false,
            "jcr:lastModified": "2019-10-09 18:13:38",
            "jcr:mimeType": "image/png",
            "jcr:mixinTypes": [
              "mix:versionable"
            ],
            "jcr:predecessors": [
              
            ],
            "jcr:primaryType": "nt:resource",
            "jcr:uuid": "1aa7c9ac-11fd-4bf5-8ffb-cba0d73bd7fc",
            "jcr:versionHistory": "9238ba45-1630-416d-8817-2063e87d0b54"
          }
        }
      }
    }
  },
  "77941d4b-e2fa-4fef-874a-97ee6e4c4765": {
    "jcr:created": "2022-03-24 20:48:10",
    "jcr:createdBy": "admin",
    "jcr:primaryType": "sling:OrderedFolder",
    "home": {
      "jcr:created": "2022-03-24 20:48:10",
      "jcr:createdBy": "admin",
      "jcr:primaryType": "sling:OrderedFolder",
      "about": {
        "jcr:created": "2022-03-24 20:48:10",
        "jcr:createdBy": "admin",
        "jcr:primaryType": "sling:OrderedFolder",
        "jcr:content": {
          "jcr:baseVersion": "60008a5a-074f-484f-9f10-9ce7f5a3f4e5",
          "jcr:isCheckedOut": false,
          "jcr:predecessors": [
            
          ],
          "jcr:primaryType": "cpp:PageContent",
          "jcr:uuid": "77941d4b-e2fa-4fef-874a-97ee6e4c4765",
          "jcr:versionHistory": "39e13ee2-5db5-4037-acd6-73380ed2ee2a"
        }
      }
    }
  }
}
```

**Comment by stoerr** [comment]
```
10.08.2023 10:39:48.125 *WARN* [0:0:0:0:0:0:0:1 [1691656788104] GET /bin/cpm/nodes/source.pkg/content/ist/composum/cpl:attic HTTP/1.1] com.composum.slin* g.nodes.servlet.SourceModel Duplicated entry for jcr_root/content/ist/composum/_cpl_attic/1aa7c9ac-11fd-4bf5-8ffb-cba0d73bd7fc/assets/general/adaptTo.pn* g - skipping duplicate
  java.util.zip.ZipException: duplicate entry: jcr_root/content/ist/composum/_cpl_attic/1aa7c9ac-11fd-4bf5-8ffb-cba0d73bd7fc/assets/general/adaptTo.png
          at java.base/java.util.zip.ZipOutputStream.putNextEntry(ZipOutputStream.java:233)
          at com.composum.sling.nodes.servlet.SourceModel.putEntry(SourceModel.java:668) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writeFile(SourceModel.java:629) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writeIntoZip(SourceModel.java:568) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writeFile(SourceModel.java:656) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writeIntoZip(SourceModel.java:568) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writeIntoZip(SourceModel.java:592) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writeIntoZip(SourceModel.java:592) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writeIntoZip(SourceModel.java:592) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writeIntoZip(SourceModel.java:592) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writeIntoZip(SourceModel.java:592) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceModel.writePackage(SourceModel.java:395) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at com.composum.sling.nodes.servlet.SourceServlet.doGet(SourceServlet.java:131) [com.composum.nodes.console:4.2.2.SNAPSHOT]
          at org.apache.sling.api.servlets.SlingSafeMethodsServlet.mayService(SlingSafeMethodsServlet.java:266) [org.apache.sling.api:2.18.4]
          at org.apache.sling.api.servlets.SlingSafeMethodsServlet.service(SlingSafeMethodsServlet.java:342) [org.apache.sling.api:2.18.4]

```

**Comment by stoerr** [comment]
That should be fixed with #78dcddcfb9e900dd0373304456aea03c935efadf and thus I'll close it.
